### PR TITLE
Test coverage with coveralls.

### PR DESCRIPTION
**Pull request by bjlittle**
Added test coverage loveliness.

### CODE REVIEW

**Comment by bjlittle** [comment]
@dkillick Fancy looking at this ...

File: .coveragerc [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#
+# .coveragerc to control coverage.py
+#
+
+[run]
+branch = True
+omit = 
+    setup.py
+    iris_grib/tests/*
```
Comment (dkillick) [inline .coveragerc]: Really?

Comment (bjlittle) [inline .coveragerc]: Yes, really. The point of testing coverage is to determine the percentage of code that is tested. 

By running the tests, coverage calculates the coverage of code that is exercised by running the tests. We don't want to dilute that statistic with the actual test code itself.

**Comment by dkillick** [comment]
Like it! :+1: for test coverage :smile:

**Pull request merged**